### PR TITLE
Improve options dialoge

### PR DESCRIPTION
**Pull request by fynngodau**
This pull request improves the options window.

Previously:

image

With changes:

image

* Changes all the labels to nominal style (making them shorter, easier to skim, better-looking)
* Removes vertical scroll bar
* Increases options window size (I didn't know why it was so small, so I used `minimumSize`)
* It appears that the option categories appear in a random order, but I don't think these changes cause that
* `Increase identity feature level to software feature level if possible` may be confusing, but so is `Update supported feature level of used identity if stored feature level is lower than the one supported by this software`

### CODE REVIEW

File: src/dialogs/OptionsDialog.ui [hunk 1]
```diff
@@ -36,12 +42,11 @@
        <rect>
         <x>0</x>
         <y>0</y>
-        <width>982</width>
-        <height>597</height>
+        <width>1051</width>
+        <height>665</height>
```
Comment (fynngodau) [inline src/dialogs/OptionsDialog.ui]: Qt Creator insisted ¯\\_(ツ)\_/¯

**Comment by blizzard4591** [comment]
The class of the option dialog has 

```
int nWidth = 300;
int nHeight = 400;
if (parent != nullptr) {
	setGeometry(parent->x() + parent->width() / 2 - nWidth / 2, parent->y() + parent->height() / 2 - nHeight / 2, nWidth, nHeight);
} else {
	resize(nWidth, nHeight);
}
```
so I guess the width/height set in the dialog is ignored in favor of the size of the main application. Whether this makes more sense stands to be discussed.

File: src/options/OptionReader.cpp [hunk 2]
```diff
@@ -69,14 +69,14 @@ namespace openmittsu {
...
-			target->registerOption(OptionGroups::GROUP_GENERAL, Options::BOOLEAN_TRUST_OTHERS, QStringLiteral("options/trustOthers"), tr("Whether to accept messages from users whos group membership has not (yet) been confirmed."), false, OptionTypes::TYPE_BOOL, OptionStorage::STORAGE_DATABASE);
-			target->registerOption(OptionGroups::GROUP_GENERAL, Options::BOOLEAN_UPDATE_FEATURE_LEVEL, QStringLiteral("options/updateFeatureLevel"), tr("Whether the supported feature level of the used identity should be updated if the stored feature level is lower than the one supported by this app."), true, OptionTypes::TYPE_BOOL, OptionStorage::STORAGE_DATABASE);
-			target->registerOption(OptionGroups::GROUP_GENERAL, Options::FILEPATH_DATABASE, QStringLiteral("options/database/databaseFile"), tr("The file path where the main database file is stored."), "", OptionTypes::TYPE_FILEPATH, OptionStorage::STORAGE_SIMPLE);
+            target->registerOption(OptionGroups::GROUP_PRIVACY, Options::BOOLEAN_SEND_TYPING_NOTIFICATION, QStringLiteral("options/sendTypingNotification"), tr("Send \"user is typing\" notifications"), true, OptionTypes::TYPE_BOOL, OptionStorage::STORAGE_DATABASE);
```
[the remark this answers is not in the document]
Comment (blizzard4591) [reply src/options/OptionReader.cpp]: @fynngodau Could you fix the indentation? Then I will merge :)

[the remark this answers is not in the document]
Comment (fynngodau) [reply src/options/OptionReader.cpp]: @blizzard4591 Fixed!

**Pull request merged**